chingu-voyages/v37-bears-team-14
# Implement github auth for User

## Manual test
1. Add new configs: https://discord.com/channels/330284646283608064/939665645107282010/940802454881194046
2. Run `yarn dev` 
3. Navigate to `localhost:3000/auth/github`
4. Check that you can authenticate with github and are redirected to "/"

Closes #23
